assets/src/styles/blocks/HubspotForm/HubspotFormThankYouStyles.scss

```diff
@@ -0,0 +1,56 @@
+@import "../../master-theme/assets/src/scss/base/variables";
+@import "../../master-theme/assets/src/scss/base/colors";
+@import "../../master-theme/assets/src/scss/base/mixins";
+@import "../../master-theme/assets/src/scss/base/fonts";
+
+.hubspot-form-thankyou {
+  margin: 48px 0;
+}
+
+.hubspot-form-thankyou .wp-block-group {
```

Comment:
Should we change this to only affect the top level WP group blocks? That way it doesn't affect any possible inner groups.